src/components/flow/routers/response/ResponseRouterForm.tsx

```diff
@@ -529,11 +529,11 @@ export default class ResponseRouterForm extends React.Component<
       isTestingAvailable || this.state.automatedTestCases[this.state.testingLang.value].length > 0;
     const tabs = [
       {
-        name: 'Testing',
+        name: 'Test Responses',
         checked: checked,
         body: this.renderTestingTab(),
         hasErrors: !checked,
-        nameStyle: checked ? '' : styles.testingTabNameError,
+        icon: !checked ? 'fe-x' : null,
```

Comment:
If it's `checked`, should not have the `check` icon?

Reply:
if checked it should will show checked icon, if not checked that is when it shows red text which I just replaced the "x"